src/tutorial/further.md

```diff
@@ -68,8 +68,8 @@ on your interests, but we recommend starting with the ones marked with a ⭐.
 - [Example 23](https://docs.mfem.org/html/ex23_8cpp_source.html) &mdash; Second-order in time wave equation. ⭐
 - [Example 24](https://docs.mfem.org/html/ex24_8cpp_source.html) &mdash; Mixed finite element spaces and interpolators.
 - [Example 25](https://docs.mfem.org/html/ex25_8cpp_source.html) &mdash; Perfectly Matched Layer (PML) for Maxwell equations.
-- [Example 26](https://docs.mfem.org/html/ex26_8cpp_source.html) &mdash; Multigrid preconditioner for the Laplace problem. ⭐
-- [Example 27](https://docs.mfem.org/html/ex27_8cpp_source.html) &mdash; Boundary conditions for the Laplace problem.
+- [Example 26](https://docs.mfem.org/html/ex26_8cpp_source.html) &mdash; Multigrid preconditioner for the Poisson problem. ⭐
+- [Example 27](https://docs.mfem.org/html/ex27_8cpp_source.html) &mdash; Boundary conditions for Laplace's equation.
```

Comment:
```suggestion
- [Example 27](https://docs.mfem.org/html/ex27_8cpp_source.html) &mdash; Boundary conditions for elliptic problems.
```
to be consistent with changes to `src/examples.md`

Reply:
I addressed this differently (to be consistent with `mfem-4.8/doc/CodeDocumentation.dox`)